### PR TITLE
[cleanup][types] Remove 6 unused, untested functions from account_config

### PR DESCRIPTION
**Pull request by huitseeker**
This PR removes 6 untested, unused functions from `types::account_config`. The value of the PR may reside more in the signaling of the nature of those untested, unused methods rather than actual removal.

I do hope & expect that reviewers will come up with valid reasons why one or another of those methods should be part of our public API — I've hence separated commits per method so that those valid methods can be kept, by omitting the commit that removes them. Just say so in review, plz :slightly_smiling_face: 

(I also have found 9 such unused, untested functions that might be removed in `language/`, 
https://gist.github.com/2f8ddffbbc18df783f1bad0b22375613
which I won't submit for a PR, as requested  [[1]](https://github.com/libra/libra/pull/3879#issuecomment-628976319) [[2]](https://github.com/libra/libra/pull/3879#issuecomment-629417720) )

### CODE REVIEW

**Comment by davidiw** [comment]
Adding @runtian-zhou, as I think he might have context. I'm planning on landing this PR end of day tomorrow (Tuesday).

**Comment by runtian-zhou** [comment]
LGTM! These are no longer in use due to the introduction of `MoveResource` trait.

File: types/src/account_config/constants/account.rs [hunk 1]
```diff
@@ -27,41 +24,10 @@ static SENT_EVENT_NAME: Lazy<Identifier> =
...
-pub fn account_struct_tag() -> StructTag {
-    StructTag {
-        address: CORE_CODE_ADDRESS,
-        module: ACCOUNT_MODULE_IDENTIFIER.clone(),
-        name: ACCOUNT_STRUCT_NAME.to_owned(),
-        type_params: vec![],
-    }
-}
-
-pub fn sent_payment_tag() -> StructTag {
```
Comment (thefallentree) [inline types/src/account_config/constants/account.rs:51]: wait, isn't this used in json-rpc crate?

Comment (huitseeker) [reply types/src/account_config/constants/account.rs:51]: Nope!

**Pull request closed without merging**